flask_resty/utils.py

```diff
@@ -52,9 +53,9 @@ def __init__(self, get_default):
     def __get__(self, instance, owner):
         if instance is None:
             return self
-        try:
+        if hasattr(instance, self.internal_field_name):
```

Comment:
Slightly more explicit rewrite

Reply:
no preference, is there any perf penalty (like dbl lookup?)

Reply:
Hmm, didn't consider but I suspect the former was actually less performant (caught exception vs key lookup)